### DISCUSSION TITLE
Performance issue with mod menu opened

### DISCUSSION THREAD

**Discussion by alexpado** [ppy/osu#24398] [Q&A] [answered]
### Type

Performance

### Bug description

When opening the mod menu, the number of frame updated & drawn are divided by two, as you can see on the video provided.

## Hardware:
- CPU: i7-9750H (2.6GHz)
- GPU: NVIDIA GeForce GTX 1660 Ti

Yes it's a laptop with 2 GPU (one from the GPU and the other from the CPU). I double checked and osu! runs on the dedicated one (GPU)

Doing the tests with or without any software running (like NVIDIA Shadowplay, Discord screenshare) doesn't affect this behavior, the performance drop stay the very same.

### Screenshots or videos

https://github.com/ppy/osu/assets/31733699/a62b4649-9d83-4b25-8799-c750b9d6dc91

### Version

57e51f4 (master)

### Logs

[network.log](https://github.com/ppy/osu/files/12195192/network.log)
[performance.log](https://github.com/ppy/osu/files/12195193/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12195194/runtime.log)
[input.log](https://github.com/ppy/osu/files/12195195/input.log)

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. Especially on android devices, where the hardware can be of varying quality, it may be impossible to run well on some devices. Please be patient and wait for future optimisations, or upgrade your device.

**Reply by alexpado** [reply]
I was just reporting it for traceability concern, not asking for an immediate fix

**Reply by peppy** [reply]
Sure. It's just many things being on screen at once.

**Comment by Minorsonek** [comment]
I second this, same issues present on M1 Max Macbook, stable 180-200fps everywhere, but when entering mods menu it goes down to like 50fps, briefly but still - enough to be noticeable

**Reply by Theighlin** [reply]
Are you using the opengl renderer? Metal is the default, and performs better overall, if you haven't tried it it recently, try switching back (even if your fps is locked to refresh rate, it's the lowest possible latency)
